src/datadreamer/image_generation/clip_image_tester.py

```diff
@@ -39,11 +42,13 @@ def test_image(self, image: Image.Image, objects: List[str], conf_threshold=0.05
         # Process the inputs for the CLIP model
         inputs = self.clip_processor(
             text=objects, images=image, return_tensors="pt", padding=True
-        )
+        ).to(self.device)
 
         # Get similarity scores from the CLIP model
         outputs = self.clip(**inputs)
-        logits_per_image = outputs.logits_per_image  # image-text similarity score
+        logits_per_image = (
+            outputs.logits_per_image.detach().cpu()
```

Comment:
should the `.cpu()` be here?

Reply:
@HonzaCuel why did you add .detach().cpu()? Shouldn't it work without this?

Reply:
@sokovninn, yeah, you're probably right; I'll try it.